projects/pastanaga-angular/src/lib/styles/_avatar.scss

```diff
@@ -0,0 +1,123 @@
+@import "./variables";
+
+
+@mixin full-size-circle() {
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+}
+
+@mixin flex-content-aligned() {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.pa-avatar {
+
+    abbr[title], acronym[title] {
+        border-bottom: none;
+        text-decoration: none;
+    }
+
+    position: relative;
+    display: flex;
+    width: rhythm($avatar-default-rhythm);
+    height: rhythm($avatar-default-rhythm);
+
+    &.pa-avatar-large {
+        width: rhythm($avatar-large-rhythm);
+        height: rhythm($avatar-large-rhythm);
+        .pa-avatar-default .pa-avatar-initials {
+            @include font-size($avatar-large-text-size);
+        }
+        .pa-avatar-user-icon {
+            width: rhythmRatio($avatar-large-rhythm, $avatar-icon-size-ratio);
+            height: rhythmRatio($avatar-large-rhythm, $avatar-icon-size-ratio);
+        }
+        .pa-avatar-icon-alt {
+            @include font-size($avatar-icon-alt-font-size);
+            font-weight: $avatar-icon-alt-font-weight;
+        }
+    }
+
+    .pa-avatar-icon {
+        @include full-size-circle();
+        @include flex-content-aligned();
+        background: $avatar-icon-default-background;
+    }
+
+    .pa-avatar-img {
+        @include full-size-circle();
+        object-fit: cover;
+    }
+
+    .pa-avatar-default {
+        display: inline-flex;
+        @include full-size-circle();
+        .pa-avatar-initials {
+            @include font-size($avatar-default-text-size);
+            margin: auto;
+            color: $avatar-initials-color;
+            font-weight: $avatar-initials-font-weight;
+            text-transform: uppercase;
+        }
+        &.pa-avatar-blue {
```

Comment:
Maybe we could create a mixing for not having to duplicate the property for each color.

Reply:
in this case, as we want to component to set the color, it is easier to set a class than calling a mixin